packages/subproviders/src/subproviders/subprovider.ts

```diff
@@ -34,9 +33,6 @@ export class Subprovider {
     }
     public setEngine(engine: any): void {
         this.engine = engine;
-        engine.on('block', (block: any) => {
-            this.currentBlock = block;
-        });
```

Comment:
Mh... we cannot remove this... Even though we don't use it, it could be used by a super class. I think this should be `protected` rather then `private`.

Reply:
Could be. Dead code. We'll add it as soon as it will be used by a super class.